migration-guides/0.15-Migration-Guide.md

```diff
@@ -276,48 +276,405 @@ parcel build index.html --no-source-maps --no-optimize --no-scope-hoist --dist-d
 parcel build index.html --no-source-maps --dist-dir "dist/" # minified bundle for the browser
 ```
 
-## Breaking Changes Made in Core Libraries
+---
 
-### Changes affecting multiple libraries
+## Changes made across all `core`, `contrib`, `web`, and `node` libraries
 
 - Migrated all FFI to ES modules and dropped support for CommonJS modules.
+- Update project and deps to PureScript v0.15.0.
+
+When applicable, the following changes were also made:
+- Removed `MonadZero` type class and all of its deprecated instances.
 - Removed all kind-specific Proxy types (e.g. `SProxy`, `Proxy2`, `Proxy3`, `RLProxy`, etc.)
     - Replace usage of such types with `Type.Proxy (Proxy(..))`.
-- Removed `MonadZero` type class and all of its deprecated instances.
+- Removed `forall proxy. proxy "a symbol" -> ...` workaround.
+
+  This workaround was needed to reduce breakage in `v0.14.0` ecosystem update
+  when polykinded `Proxy` was originally introduced.
+
+## `purescript` Libraries
+
+### purescript-arrays
+
+Breaking changes:
+- Drop deprecated `group'` and `empty` (#219 by @JordanMartinez)
+
+Other improvements:
+- Fixed minor documention issue with `find` (#216 by @JamieBallingall)
+
+### purescript-console
+
+New features:
+- Added `debug` (#36)
+
+### purescript-control
+
+Breaking changes:
+- Make `<|>` right associative (#80 by @JordanMartinez)
```

Comment:
This, for example, does make sense to me to stay in the migration guide. But we aren’t explaining what to do about it at all — if we’re going to include a change in the guide then we should tell you what you might need to do. For example, this change could mention that it led to changes in associativity or precedence for quickcheck and string-parsers and parsers.

If you look at last year’s migration guide you can see that the changes aren’t just a list — they include why the change was made and what you should do about it